content/src/fieldConfig/business-formation.json

```diff
@@ -153,7 +154,10 @@
         "tableHeader": "Name,Address,Sign*,Actions",
         "titleLabel": "Title",
         "description": "Authorized Representative(s) should sign below by adding their full name. You can have up to 10 signers listed.",
-        "errorInlineAdditionalSignerName": "Enter the first and last name of each signer."
+        "errorInlineAdditionalSignerName": "Enter the first and last name of each signer.",
+        "errorInLineNameAndSignatureMissing": "Enter a signer’s name and click the checkbox to authorize their signature.",
+        "errorInLineNameMissing": "Enter a signer’s name.",
```

Comment:
I see that the copy is from the design, but we already have `"errorInlineFirstSignerName": "Enter a signer.",` in our code? Are we sure the Content Team wants to add this?

[pebble] OAN: If we are keeping this new copy, can we change it to `errorInlineAdditionalSignerName`, omitting the `Missing` and having `SignerName` as our pattern (as mentioned previously)?

Reply:
Let me double check with content!